addons/l10n_fi/static/description/index.html

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should be in the manifest, not here.

Reply:
The xml header or the content of the file ?

Reply:
moved to manifest

Reply:
__manifest__.py    --> we don't do static/description/index.html normally